### ISSUE TITLE
CPU not recognized with Ubuntu system

### ISSUE THREAD

**Issue by 11Ki11** [profesorfalken/jSensors#32]
I have added JSensors to my program to determine the CPU load of my system. It worked on Windows, but seems to show way too high values. When I ran my program in an Ubuntu system, it did not get any values at all and did not detect my CPU.
Is there a configuration difference between Windows and Linux that I need to be aware of?

Thanks

**Comment by jrddunbr** [comment]
Yes. There are different libraries in use that will provide different results.

On Windows, this program is using the OpenHardwareLib.dll file (provided by https://openhardwaremonitor.org/)

On Linux, this program is using the libsensors.so file (provided by https://github.com/lm-sensors/lm-sensors)

Notably, libsensors.so might require opting into detecting certain devices. Typically, this can be done with `sudo sensors-detect` for the `sensors` program if `lm_sensors` is installed on your Linux system. It's unclear if using libsensors.so opts you into those features or not.
